core/whistle_amqp-1.0.0/src/wh_amqp_assignments.erl

```diff
@@ -149,6 +149,11 @@ handle_call({'request_float', Consumer, Broker}, _, State) ->
     {'reply', assign_or_reserve(Consumer, Broker, 'float'), State};
 handle_call({'request_sticky', Consumer, Broker}, _, State) ->
     {'reply', assign_or_reserve(Consumer, Broker, 'sticky'), State};
+handle_call({'release_assignments', Consumer}, _, State) ->
```

Comment:
If you are converting the cast to call please either remove the handle_cast for 'release_assignments' if it is not used or update it to behave the same way.